src/library/Uno.Material/Styles/Controls/Card.xaml

```diff
@@ -1,23 +1,23 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
maybe this file isnt supposed to be committed?

Reply:
This is intentional to keep namespace consistency throughout application